tests/backends/test_cursors.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+
+
+class LoggingCursorMixin:
+    def __init__(self, bucket, *args, **kwargs):
+        super().__init__(bucket, *args, **kwargs)
+        self.bucket = bucket
+
+    def execute(self, sql, *args, **kwargs):
+        self.bucket.append(sql)
+        super().execute(sql, *args, **kwargs)
+
+
+@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL specific test.')
+class MySQLCursorOptionsTestCase(TestCase):
+    import MySQLdb
+    from MySQLdb.cursors import Cursor
```

Comment:
Sorry for the confusion. I mistook the `try ... except` as being in test methods.
Yes, you should `try` the import. Also, you'll need to define `Cursor` in case of `ImportError` to make the rest of the code syntactically correct. Maybe something like this:
```
    try:
        from MySQLdb.cursors import Cursor
    except ImportError:
        class Cursor:
            pass
```